eventbus/driver/nats.go

```diff
@@ -439,8 +446,9 @@ func (mh *eventSourceMessageHolder) ackAndCache(m *stan.Msg, id string) {
 func (mh *eventSourceMessageHolder) reset(depName string) {
 	mh.parameters[depName] = false
 	delete(mh.msgs, depName)
+
 	if mh.isCleanedUp() {
-		mh.setLastResetTime(0)
+		mh.resetTimeout = 0
```

Comment:
This should be protected by a lock, e.g. we could utilize `setLastResetTime()`.

Reply:
okay, fixed this